server.js

```diff
@@ -47,6 +48,21 @@ if (process.env.FORCE_HTTPS === "true") {
 	app.use(enforce.HTTPS({trustProtoHeader: true}));
 }
 
+if (process.env.IP_WHITELIST) {
+        let clientIp = function(req, res) {
+                return req.headers['x-forwarded-for'] ? (req.headers['x-forwarded-for']).split(',').pop() : ""
```

Comment:
This should indeed work for heroku deployment, but can't validate locally well because there is no x-forwarded-for header set.  I think this should change to look at the host header if x-forwarded-for is not set.  And include 'localhost' in the whitelist.

Reply:
Good point, I had assumed this was Heroku only.

I'll update it to be:
if(x-forwarded-for header) return last entry in x-forwarded-for header
else use tcp/ip remote_ip